insights_content_service_test.sh

```diff
@@ -23,6 +23,16 @@ fi
 #set NOVENV is current environment is not a python virtual env
 [ "$VIRTUAL_ENV" != "" ] || NOVENV=1
 
+function install_certificates() {
+    if [ -z "$ENV_DOCKER" ]; then
+        return
+    fi
+
+    curl -ksL https://certs.corp.redhat.com/certs/2022-IT-Root-CA.pem -o /etc/pki/ca-trust/source/anchors/2022-IT-Root-CA.pem
+    curl -ksL https://certs.corp.redhat.com/certs/Current-IT-Root-CAs.pem -o /etc/pki/ca-trust/source/anchors/Current-IT-Root-CAs.pem
```

Comment:
I think you only need this one.

Reply:
Yes,and it didn't solve the problem because the scripts are not run with su-permissions